src/js/components/TimeFieldComponent.jsx

```diff
@@ -12,18 +12,20 @@ var TimeFieldComponent = React.createClass({
   render: function () {
     var time = this.props.time;
 
+    var timeField = time != null
+      ? (
+          <dd>
+            <time dateTime={time}>
+              {new Date(time).toLocaleString()}
+            </time>
+          </dd>
+        )
+      : <dd className="text-muted">None</dd>;
```

Comment:
Yes, this is better.

Actually I did it completely wrong before, because we have an unwritten rule, that we don't use ternaries for markup like this, only for simple value differentiation. 

Will add it to the guidelines.